examples/flava/native/configs/10b.yaml

```diff
@@ -0,0 +1,81 @@
+training:
```

Comment:
do u want to check in the 4.8b one while you are at it

Reply:
Sure!